src/containers/Searchbar.js

```diff
@@ -14,7 +14,16 @@ const SearchBar = ({ searchBooks }) => {
   return (
     <SearchWrapper>
       <InputGroup>
-        <Input placeholder="Search for books you nerd..." id="search" />
+        <Input
+          autofocus="true"
+          onKeyPress={event => {
```

Comment:
maybe  move this function logic out of here. Also, in general its better to avoid using anonymous functions in prop ;)